eslint.config.mjs

```diff
@@ -264,6 +264,18 @@ export default [
         }
       ],
       'import/no-self-import': 'error',
+      'no-restricted-imports': [
+        'error',
+        {
+          patterns: [
+            {
+              group: ['node:fs', 'fs'],
+              message:
```

Comment:
just had an idea.  While we're here, let's also ban `fs-extra` so that never comes back in accidentally.